src/main/kotlin/no/nav/familie/ef/søknad/integration/PdlClient.kt

```diff
@@ -24,22 +24,27 @@ class PdlClient(val pdlConfig: PdlConfig,
         val pdlResponse: PdlResponse<PdlSøkerData> = postForEntity(pdlConfig.pdlUri,
                                                                    pdlPersonRequest,
                                                                    httpHeaders())
-        return feilsjekkOgReturnerData(personIdent, pdlResponse).person
+        return feilsjekkOgReturnerData(personIdent, pdlResponse) { it?.person }
```

Comment:
trenger vi egentlig `?` her? Kaster jo exception hvis den ikke er satt?

Reply:
> Eksempel på tilfelle hvor man har bedt om flere elementer (fornavn, etternavn og bostedsadresse), men hvor det har skjedd en feil ved uthenting av bostedsadresse. Her vil GraphQL levere den dataen den har tilgjengelig.

Dette er vel et scenario vi ikke støtter. Får vi "litt" data, men ikke alt vi krever så vil vi få litt trøbbel selv om vi har nullable på pdlSøker? Men jeg vet ikke hvordan... 

Kanskje en try/catch rundt `postForEntity` som kan parse error-verdien uten at vi prøver å parse json-verdien. Men det er kanskje for "sent" hvis den allerede har prøvd å parse dataene og feilet fordi noe er mangelfullt?

Reply:
(denne diskusjonen ble tatt videre på slack https://nav-it.slack.com/archives/G012YEK9YQ1/p1606380824386200 )